nix/modules/telegrafd.nix

```
@@ -0,0 +1,100 @@
+{ pkgs, lib, config, ... }: {
+  systemd.services.telegraf.path = [ pkgs.nvme-cli ];
+
+  services.telegraf = {
+    enable = true;
+    extraConfig = {
+      agent.interval = "60s";
+      inputs = {
+        prometheus.urls = [
+          "http://localhost:3030/metrics"
```

We also need to monitor kuutamod and the near-exporter (I don't know if have the near exporter yet in the opensource repo, but it's needed for the dashboard).
```suggestion
          "http://localhost:3030/metrics"
          "http://localhost:2233/metrics"
          ] ++ lib.mapAttrsToList (_: inst: "http://${inst.host}:${inst.port}")
            config.services.near-prometheus-exporter.instances;
```

---

You can move it from our internal repository to this one.